Editor/Tasks/AbstractProcessTask.cs

```diff
@@ -9,17 +11,27 @@ internal abstract class AbstractProcessTask : ITask
 	{
 		private static readonly string projectDirectory;
 
+		public bool ReceivedError
```

Comment:
`RecievedError` sounds like a method name, maybe `didReceiveError`, R

Reply:
Agreed although to me it's not a method name, ReceiveError() would be. I already changed it on the next branch